pkg/operation/botanist/logging.go

```diff
@@ -70,3 +95,16 @@ func (b *Botanist) DeploySeedLogging(ctx context.Context) error {
 
 	return b.K8sSeedClient.ChartApplier().Apply(ctx, filepath.Join(charts.Path, "seed-bootstrap", "charts", "loki"), b.Shoot.SeedNamespace, fmt.Sprintf("%s-logging", b.Shoot.SeedNamespace), kubernetes.Values(lokiValues))
 }
+
+func (b *Botanist) isShootNodeLoggingEnabled() bool {
+	if b.Shoot != nil && b.Shoot.IsLoggingEnabled() && b.Config != nil &&
+		b.Config.Logging != nil && b.Config.Logging.ShootNodeLogging != nil {
+
+		for _, purpose := range b.Config.Logging.ShootNodeLogging.ShootPurposes {
+			if gardencore.ShootPurpose(b.Shoot.Purpose) == purpose {
+				return true
+			}
+		}
+	}
+	return false
+}
```

Comment:
If `b.Config == nil` or `b.Config.Logging == nil` or `b.Config.ShootNodeLogging == nil` then I would expect that the node-logging feature is enabled for the shoot? Why do I have to provide some configuration for purposes to enable it? I simply want it to be enabled for all purposes by default.

OTOH, if you decide that it's "too dangerous" to enable it by default for all purposes then I would expect some document that explains this for operators/admins.

Reply:
As we were concerned about the network cost we decided to disable it by default. So we can enable it via landscape-setup and see which component what network traffic is generating. And base on that information we can remove reduce or totally remove the useless logs.

Reply:
OK, thanks! Then just some docs about the behaviour would be nice.

Reply:
I have added a .md file under `/docs/deployment` but I am not sure is that the right place.